modules/internal/ChapelRange.chpl

```diff
@@ -144,7 +144,8 @@ module ChapelRange {
   
     var _low: idxType = 1;                         // lower bound
     var _high: idxType = 0;                        // upper bound
-    var _stride: strType = 1;                      // signed stride of range
+    var _stride: if stridable then strType else void =
+                   if stridable then 1 else _void; // signed stride of range
```

Comment:
I was able to just use `void` in https://github.com/chapel-lang/chapel/commit/0b6db1aaa7c11be4b0df9cb55486b4f79da77a6a#diff-86f9faa2b8181d9c22f3b157cd767397R51